packages/orchestration/src/examples/send-anywhere.flows.js

```diff
@@ -59,26 +80,43 @@ export const sendIt = async (
   const sharedLocalAccount = await sharedLocalAccountP;
   await localTransfer(seat, sharedLocalAccount, give);
 
-  void log(`completed transfer to localAccount`);
+  if (typeof info.cctpDestinationDomain !== 'number') {
```

Comment:
ChainHub should take care of enforcing the type, so consider this more succinct approach:

```suggestion
  if (!('cctpDestinationDomain' in info)) {
```

Reply:
done

Reply:
done